src/app/globals.css

```
@@ -164,14 +164,25 @@
   }
   body {
     @apply bg-background text-foreground;
+    background-image: repeating-linear-gradient(
+      90deg,
+      transparent 0px,
+      transparent 2px,
+      rgba(139, 122, 104, 0.035) 2px,
+      rgba(139, 122, 104, 0.035) 3px
+    );
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)